CONTRIBUTING.md

```diff
@@ -68,6 +68,16 @@ If successful, it will modify `activities.json` with the new specification. Chec
 }
 ```
 
+### Asking Mozilla to Update a Position
+
+Specifications might be updated after a position is decided. Especially if those
+updates are in response to feedback provided by Mozilla.
```

Comment:
This "especially" should be attached to something.  I don't think it's that specifications are updated especially when Mozilla provides feedback, though. :-)  I think it's something about updating the position.